win_build/worker_vs2019.vcxproj

```diff
@@ -92,17 +92,8 @@
     <CodeAnalysisRuleAssemblies Condition="'$(Configuration)|$(Platform)'=='Release|x64'" />
     <CodeAnalysisRuleAssemblies Condition="'$(Configuration)|$(Platform)'=='Release|ARM64'" />
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
-    <TargetName>worker_windows_x86_64</TargetName>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM64'">
-    <TargetName>worker_windows_x86_64</TargetName>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
-    <TargetName>worker_windows_x86_64</TargetName>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|ARM64'">
-    <TargetName>worker_windows_x86_64</TargetName>
+  <PropertyGroup >
+    <TargetName>$(ProjectName)_windows_$(Platform)</TargetName>
```

Comment:
This will change architecture name from `x86_64` to `x64`. For compatibility target name architecture should be `x86_64`.

Reply:
The architecture is x64.

Reply:
And should be `x86_64`

Reply:
And should be `x64`